lib/kitchen/directions/bake_appendix.rb

```diff
@@ -18,6 +18,7 @@ def self.v1(page:, number:)
           title = section.first!("[data-type='title']")
           title.name = "h#{section['data-depth'].to_i + 1}"
         end
+        page
```

Comment:
what's the thinking on this return value?

Reply:
I was thinking about return types and how it didn’t make much sense for this function to return what it previously did, which was an integer (I believe?) — I can take out this change, or change to nil or a different value — what do you think? (Does it matter?)

Reply:
I've reverted this